ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/offset/LatestByOffsetUdafTest.java

```diff
@@ -45,19 +45,48 @@ public void shouldInitialize() {
     assertThat(init, is(notNullValue()));
   }
 
+  @Test
```

Comment:
Apply same changes to this file as were made to the `EarliestByOffsetTest` file.